vip-helpers/sandbox.php

```diff
@@ -88,3 +88,21 @@ function wpcom_do_sandbox_bar() {
 	endif;
 }
 add_filter( 'wpcom_show_sandbox_bar', '__return_true' );
+
+/**
+ * Filters plugin URLs to use sandboxed hostnames.
+ *
+ * @param string $url The complete URL to the plugins directory including scheme and path.
+ *
+ * @return string Filtered URL using sandboxed hostname.
+ */
+function wpvip_filter_sandbox_plugins_url( $url ) {
+	if ( defined( 'WPCOM_SANDBOXED' ) && WPCOM_SANDBOXED ) {
```

Comment:
We don't need this check since the file returns if the constant isn't defined. Other than that, tested and LGTM.

Reply:
I think this should need to be swapped with the check for the old platform?